texts/story.uzb.txt

```diff
@@ -0,0 +1,11 @@
+1	BAXTIYOR QAYERDA?
+2	Baxtiyor va Oygul bog’da. Bugun ob-havo yaxshi, havo juda iliq. Ammo kecha juda sovuq edi! Ular o’ynay olishmadi. Baxtiyor va Oygul o’ynashni yaxshi ko’radilar, ular har doim katta hovli oldidagi bog’da birga o’ynashadi.
```

Comment:
You should probably correct these to use `ʻ` instead of `’`, but it doesn't matter that much.